.github/workflows/devbuild.yml

```diff
@@ -552,3 +552,61 @@ jobs:
           cmake --build ${{ github.workspace }}/build `
             --config ${{ matrix.cmake_build_type }} `
             --target run_viewer_pytest
+
+      - name: generate portable
+        if: ${{ matrix.cmake_build_type == 'Release' }}
+        run: |
+          # Get the Python version installed and extract the major+minor version components
```

Comment:
It is good enough to have the commands listed in the Github Actions yaml file for now.  In the future, would it be useful to absorb the commands to a standalone script file.  Or even make a Python module to do it?  (This is only for discussion.  No action is needed for this PR.)

Reply:
Sure, this is a great suggestion. Integrating these commands into a standalone script or Python module would make on-premise development and testing more competent.
Thanks, I'll consider this for future improvements.